src/main/java/io/strimzi/kafka/bridge/quarkus/RestBridge.java

```diff
@@ -548,7 +565,53 @@ private boolean isReady() {
         return this.isReady;
     }
 
-    private void startInactiveConsumerDeletionTimer(Long timeout) {
-        // TODO: to be used a Quarkus timer for scheduling the task
+    private void startInactiveConsumerDeletionTimer(long timeout) throws SchedulerException {
```

Comment:
let's use the plural and change to `scheduleInactiveConsumersDeletionJob`, wdyt?